source/includes/fundamentals/code-examples/connection/AtlasConnection.cs

```diff
@@ -1,11 +1,18 @@
 using MongoDB.Driver;
+using MongoDB.Bson;
 
-// Connection URI
+// Replace the placeholders with your credentials
 const string connectionUri = "mongodb+srv://<username>:<password>@cluster0.sample.mongodb.net/?retryWrites=true&w=majority";
 
-var serverApi = new ServerApi(ServerApiVersion.V1);
 var settings = MongoClientSettings.FromConnectionString(connectionString);
+
+// Set the ServerApi field of the settings object to Stable API version 1
+var serverApi = new ServerApi(ServerApiVersion.V1);
 settings.ServerApi = serverApi;
 
 // Create a new client and connect to the server
-var client = new MongoClient(settings);
\ No newline at end of file
+var client = new MongoClient(settings);
+
+// Send a ping to confirm a successful connection
+var result = client.GetDatabase("admin").RunCommand<BsonDocument>(new BsonDocument("ping", 1));
+Console.WriteLine(result.ToJson());
```

Comment:
Question:
I thought that the "successful confirmation" was going to be uniform as requested as mentioned in [this thread](https://github.com/mongodb/docs-csharp/pull/80#discussion_r1115970166). Or am I interpreting that response incorrectly?

Suggestion:
I think it would make sense to print a human-readable message that it was successful similar to [the one you added to the Go driver docs](https://github.com/mongodb/docs-golang/blob/master/source/includes/fundamentals/code-snippets/srv.go#L38) regardless of what operation is used to determine the connection.